### ISSUE TITLE
Completion not working for imported module

### ISSUE THREAD

**Issue by pubudu91** [ballerina-platform/ballerina-lang#26523]
Tried the main function BBE, with the cursor positioned as depicted below.
```ballerina
import ballerina/io;

public function main(string name,
                     int age = 18,
                     string year = "Freshman",
                     string... modules)
                  returns error? {

    // Return an error if the name is invalid.
    if (name.length() < 5) {
        error e = error("InvalidName", message = "invalid length");
        io:<cursor>
        return e;
    }

    string info = string `Name: ${name}, Age: ${age}, Year: ${year}`;

    if (modules.length() > 0) {
        info += ", Module(s): " + modules.toString();
    }
    io:println(info);
}
```
Error log:
```
[Error - 5:16:28 PM] Operation 'text/completion' failed! {uri: '/home/pubudu/wso2/ballerina-distribution/examples/the-main-function/the_main_function.bal', line: 17, col: 12, error: 'java.lang.NullPointerException'}
java.lang.NullPointerException
	at org.ballerinalang.langserver.completions.builder.TypeCompletionItemBuilder.setMeta(TypeCompletionItemBuilder.java:94)
	at org.ballerinalang.langserver.completions.builder.TypeCompletionItemBuilder.build(TypeCompletionItemBuilder.java:56)
	at org.ballerinalang.langserver.completions.providers.AbstractCompletionProvider.lambda$getCompletionItemList$0(AbstractCompletionProvider.java:171)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
	at org.ballerinalang.langserver.completions.providers.AbstractCompletionProvider.getCompletionItemList(AbstractCompletionProvider.java:154)
	at org.ballerinalang.langserver.completions.providers.context.BlockNodeContextProvider.getCompletions(BlockNodeContextProvider.java:78)
	at org.ballerinalang.langserver.completions.providers.context.BlockStatementNodeContext.getCompletions(BlockStatementNodeContext.java:40)
	at org.ballerinalang.langserver.completions.providers.context.BlockStatementNodeContext.getCompletions(BlockStatementNodeContext.java:31)
	at org.ballerinalang.langserver.completions.util.CompletionUtil.route(CompletionUtil.java:121)
	at org.ballerinalang.langserver.completions.providers.context.TypedBindingPatternNodeContext.getCompletions(TypedBindingPatternNodeContext.java:47)
	at org.ballerinalang.langserver.completions.providers.context.TypedBindingPatternNodeContext.getCompletions(TypedBindingPatternNodeContext.java:33)
	at org.ballerinalang.langserver.completions.util.CompletionUtil.route(CompletionUtil.java:121)
	at org.ballerinalang.langserver.completions.util.CompletionUtil.getCompletionItems(CompletionUtil.java:79)
	at org.ballerinalang.langserver.BallerinaTextDocumentService.lambda$completion$1(BallerinaTextDocumentService.java:177)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1700)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1692)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
```

**Comment by pubudu91** [comment]
Closing this since apparently this is the same as #26483

**Comment by nadeeshaan** [comment]
Similar to #26483 and this is fixed by #26496. Hence closing the issue
